browser/src/UI/components/Error.tsx

```diff
@@ -78,24 +83,41 @@ export class ErrorMarker extends React.Component<IErrorMarkerProps, void> {
 
     public render(): JSX.Element {
 
-        const positionDivStyles = {
+        const iconPositionStyles = {
             top: this.props.y.toString() + "px",
         }
+        const textPositionStyles = {
+            left: this.props.x.toString() + "px",
+            // Tooltip below line: use top so text grows downward when text gets longer
+            // Tooltip above line: use bottom so text grows upward
+            top: this.props.showTooltipTop ? "initial" : this.props.y.toString() + "px",
+            bottom: this.props.showTooltipTop ? "calc(100% - " + this.props.y.toString() + "px" : "initial",
+            borderColor: this.props.color,
+        }
 
-        let className = this.props.isActive ? "error-marker active" : "error-marker"
-
-        const errorDescription = this.config.getValue("editor.errors.slideOnFocus") ? (<div className="error">
-            <div className="text">
-                {this.props.text}
-            </div>
-        </div>) : null
-
-        return <div style={positionDivStyles} className={className}>
-            {errorDescription}
+        const className = [
```

Comment:
Minor suggestion here - you can use the `classnames` (https://github.com/JedWatson/classnames) module to make this a bit easier, without needing the `join(" ")`. We already have it in the project, the 'oni\browser\src\UI\components\LiveEvalMarker.tsx` uses it